apps/toolbox.fr.i18n

```diff
@@ -514,3 +514,17 @@ HartreeConstantTag = "Constante de Hartree"
 MagneticFluxQuantumTag = "Quantum de Flux Magnétique"
 ConductanceQuantumTag = "Quantum de Conductance"
 CirculationQuantumTag = "Quantum de Circulation"
+Logic = "Logic"
+LogicalBitClear = "Effacer le bit numéro b dans a"
+LogicalBitFlip = "Inverser le bit numéro b dans a"
+LogicalBitGet = "Obtenir le bit numéro b dans a"
+LogicalBitSet = "Allumer le bit numéro b dans a"
```

Comment:
Allumer is not the right word here. I'd use something like mettre/définir

Reply:
Changed 'Allumer' to 'Mettre'